salt/metalk8s/metalk8s-ui/deployed.sls

```diff
@@ -31,3 +46,16 @@ Create metalk8s-ui service:
         type: NodePort
   require:
     - pkg: Install Python Kubernetes client
+
+Create ui-api-configmap ConfigMap:
+  kubernetes.configmap_present:
+    - name: ui-api-configmap
```

Comment:
`metalk8s-ui`. We know is't a ConfigMap, and it's not necessarily only related to the API.